### PR TITLE
Hidden launcher shortcut

### PR DESCRIPTION
**Pull request by Dzakus**
https://github.com/rovo89/XposedInstaller/blob/master/src/de/robv/android/xposed/installer/ModulesFragment.java#L167-L185

### CODE REVIEW

**Comment by zst123** [comment]
I think placing the settings only launchable in Xposed Installer would be too troublesome to access.
(I'll keep this open till I am certain if I should/shouldn't commit this)

File: AndroidManifest.xml [hunk 1]
```diff
@@ -22,13 +22,14 @@
         <meta-data
             android:name="xposeddescription"
             android:value="Small Collection of Unique Features (zst123)" />
+
         <activity
             android:name="com.zst.xposed.xuimod.SettingActivity"
             android:label="@string/app_name" >
```
Comment (Dzakus) [inline AndroidManifest.xml:28]: exported="true"

**Comment by ghost** [comment]
Dzakus your ideas are shit and you should be shit

**Comment by Dzakus** [comment]
Thanks a lot. :+1:

**Comment by zst123** [comment]
Manually Implemented.

**Pull request closed without merging**